tonic/Cargo.toml

```diff
@@ -51,6 +51,7 @@ channel = [
 # harness = false
 
 [dependencies]
+async-rdma = { version = "0.4", path = "../async-rdma"}
```

Comment:
`async-rdma` is published, so `async-rdma = "0.4"` is ok.
Or you can use `async-rdma = {git = "https://github.com/datenlord/async-rdma", branch = "master"}` in the development phase to use the latest unpublished features.